unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/AssetManager/GLTF/AssetPromiseKeeper_GLTF.cs

```diff
@@ -4,9 +4,40 @@ public class AssetPromiseKeeper_GLTF : AssetPromiseKeeper<Asset_GLTF, AssetLibra
     {
         private static AssetPromiseKeeper_GLTF instance;
         public static AssetPromiseKeeper_GLTF i { get { return instance ??= new AssetPromiseKeeper_GLTF(); } }
-        
+
         public AssetPromiseKeeper_GLTF() : base(new AssetLibrary_GLTF()) { }
 
-        protected override void OnSilentForget(AssetPromise_GLTF promise) { promise.OnSilentForget(); }
+        protected override void OnSilentForget(AssetPromise_GLTF promise)
+        {
+            promise.OnSilentForget();
+        }
+
+        public override AssetPromise_GLTF Forget(AssetPromise_GLTF promise)
```

Comment:
Please let's not add a new band aid and try to fix the issue in the original `Forget` implementation. If we need to do this because we are expanding the "hack" of not being able to fix the cancelled promises, this is wrong and we need to revisit the core of the issue to make the promises truly cancellable. 

Band aids over band aids are what usually make a system no more maintainable to the point that everything has to be implemented from scratch.

Reply:
Agree. But to make promises fully cancellables we should refactor a lot of the GLTF plugin and right now we are having a lot of issues regarding GLTF handling in prod. So maybe Band aids is not an entirely wrong decision for now 🤷‍♂️

Reply:
As this is unit-tested, we agreed offline to give this solution a pass. We should very soon remove this code and implement proper cancelling for the `GLTFSceneImporter`.